test/pdo_sqlsrv/pdo_lastInsertId.phpt

```diff
@@ -0,0 +1,58 @@
+--TEST--
+Test the PDO::lastInsertId() method.
+--SKIPIF--
+--FILE--
+<?php
+  
+require_once("autonomous_setup.php");
+   
+try 
+{         
+	$database = "tempdb";
+    $conn = new PDO("sqlsrv:Server=$serverName;Database=$database", $username, $password);
+	
+	$conn->exec("IF OBJECT_ID('tmp_table1', 'U') IS NOT NULL DROP TABLE [tmp_table1]");	
+	$conn->exec("IF OBJECT_ID('tmp_table2', 'U') IS NOT NULL DROP TABLE [tmp_table2]");	
+	$conn->exec("IF OBJECT_ID('tmp_table3', 'U') IS NOT NULL DROP TABLE [tmp_table3]");	
+	$conn->exec('CREATE TABLE tmp_table1(id INT IDENTITY(100,2), val INT)');
+	$conn->exec('CREATE TABLE tmp_table2(id INT IDENTITY(200,2), val INT)');
+	$conn->exec('CREATE TABLE tmp_table3(id INT, val INT)');
+	
+	$conn->exec('INSERT INTO tmp_table1 VALUES(1)');
+	$conn->exec('INSERT INTO tmp_table2 VALUES(2)');
+	$id = $conn->lastInsertId();
+	var_dump($id);
+	$id = $conn->lastInsertId('tmp_table1');
+	var_dump($id);
+	
+	$conn->exec('INSERT INTO tmp_table2 VALUES(3)');
+	$conn->exec('INSERT INTO tmp_table1 VALUES(4)');
+	$id = $conn->lastInsertId();
+	var_dump($id);
+	$id = $conn->lastInsertId('tmp_table2');
+	var_dump($id);
```

Comment:
What's the status with lastInsertId()? Aren't we going to drop taking table names are input?